### ISSUE TITLE
Remove classify_one in favour of an option

### ISSUE THREAD

**Issue by whitfin** [fredwu/simple_bayes#2]
This is minor but it seems as though allowing `classify_one` or `classify` is unnecessary. There could just be an option of `:top` (or something better) to specify how many to classify:

``` elixir
bayes |> SimpleBayes.classify("Maybe green maybe red but definitely round and sweet.", top: 3) # classify 3

bayes |> SimpleBayes.classify("Maybe green maybe red but definitely round and sweet.") # classify all

bayes |> SimpleBayes.classify("Maybe green maybe red but definitely round and sweet.", top: 1) # classify 1, rather than a separate function
```

Seems that being generic to the number to return would be handy, and trim down any special casing.

**Comment by fredwu** [comment]
I think this is a good idea, I'll see what I can do. :)

**Comment by fredwu** [comment]
Just released v0.9.0 with this feature, please check it out. :)
